<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateCheckerClass</class>
 <widget class="QDialog" name="UpdateCheckerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update available</string>
  </property>
  <property name="windowIcon">
   <iconset resource="UpdateChecker.qrc">
    <normaloff>:/icons/software-update-available.ico</normaloff>:/icons/software-update-available.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="laterButton">
     <property name="text">
      <string>Remind me later</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="goButton">
     <property name="text">
      <string>Go to website</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="skipButton">
     <property name="text">
      <string>Skip this version</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>A newer version of Equalizer APO is available:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="AutoSizeTextEdit" name="versionBrowser">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AutoSizeTextEdit</class>
   <extends>QTextEdit</extends>
   <header>AutoSizeTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="UpdateChecker.qrc"/>
 </resources>
 <connections/>
</ui>
